src/Orchard.Web/Modules/Orchard.Taxonomies/Models/TermPart.cs

```diff
@@ -51,6 +51,11 @@ public class TermPart : ContentPart<TermPartRecord> {
             set { Store(x => x.Weight, value); }
         }
 
+        public string FullWeight {
```

Comment:
Changed. The way to do this is just setting and storing the property on the Record so Retrieve and Store doesnt save the property in the infoset, am I right?